client/src/pages/HomePage/HomePageContext.tsx

```diff
@@ -13,7 +16,23 @@ type UseHomePageContext = {
 }
 
 export const useHomePageContext = (): UseHomePageContext => {
-  const { loading, response } = useGetData({ action: getStats, params: { days: DEFAULT_DAYS } });
+  const [response, setResponse] = useState<StatsResponse | null>(null);
+  const [loading, setLoading] = useState(false);
```

Comment:
do we need loading? if yes, I think useInterval should return something like  useGetData